datasets/doc/source/conf.py

```diff
@@ -26,13 +26,15 @@
 
 
 # Fixing path issue for autodoc
-sys.path.insert(0, os.path.abspath("../../src/py"))
+sys.path.insert(0, os.path.abspath("../../"))
+sys.path.insert(0, os.path.abspath("../../flwr_datasets"))
+sys.path.insert(0, os.path.abspath("../../flwr_datasets/federated_dataset"))
```

Comment:
Are these two necessary?

Reply:
Now, I removed the bottom one, and everything works just fine.

Reply:
And how the one ending in `flwr_datasets`? The previous version seems to only point at the directory, not the package itself

Reply:
When the `sys.path.insert(0, os.path.abspath("../../flwr_datasets"))` is removed then I have as shorter API Reference
<img width="333" alt="image" src="https://github.com/adap/flower/assets/51029327/7d23f848-ca5b-4523-a5a8-1d8cc82e3bf0">

The full reference:
<img width="452" alt="image" src="https://github.com/adap/flower/assets/51029327/ff81e994-6642-467f-9601-393b9e465b4b">

Reply:
In any cases when I keep just one of them doesn't work (to produce the full API ref). The only setting that works is the 
```
sys.path.insert(0, os.path.abspath("../../"))
sys.path.insert(0, os.path.abspath("../../flwr_datasets"))
```